js/ide.js

```diff
@@ -129,19 +129,28 @@ var ide = new(function() {
 
     ide.waiter.addInfo("initialize page");
     // init page layout
-    if (settings.editor_width != "") {
+    var isInitialAspectPortrait = $(window).width() / $(window).height() < 1;
```

Comment:
hm… I'd rather switch to portrait mode at a bit smaller aspect ration, maybe 0.75 or so. (for square-ish browser windows the default layout works better)